INSTALL.md

```diff
@@ -163,6 +163,11 @@ cp config/examples/* config/
 
 #### Bundle
 
+On MacOS 10.15 first configure the bundle config with:
+```shell
+bundle config --global build.mysql2 "--with-opt-dir=/usr/local/opt/openssl"
```

Comment:
Good :+1: But I think that we also should remove what comes right after this speaking about this error.

Reply:
But that is still valid for MacOS 10.13-10.14 o?

Reply:
ah, right, sorry!